launcher/java/JavaUtils.cpp

```diff
@@ -440,18 +441,26 @@ QStringList getMinecraftJavaBundle()
 {
     QString partialPath;
     QString executable = "java";
+    QStringList processpaths;
 #if defined(Q_OS_OSX)
     partialPath = FS::PathCombine(QDir::homePath(), "Library/Application Support");
 #elif defined(Q_OS_WIN32)
     partialPath = QProcessEnvironment::systemEnvironment().value("LOCALAPPDATA", "");
     executable += "w.exe";
+
+    // add the following to the search
+    // C:\Users\USERNAME\AppData\Local\Packages\Microsoft.4297127D64EC6_8wekyb3d8bbwe\LocalCache\Local\runtime
+    auto minecraftInstaltionPath =
+        FS::PathCombine(QFileInfo(partialPath).absolutePath(), "Local", "Packages", "Microsoft.4297127D64EC6_8wekyb3d8bbwe");
+    minecraftInstaltionPath = FS::PathCombine(minecraftInstaltionPath, "LocalCache", "Local", "runtime");
+    processpaths << minecraftInstaltionPath;
```

Comment:
```suggestion

    // add the microsoft store version of the launcher to the search. the current path is:
    // C:\Users\USERNAME\AppData\Local\Packages\Microsoft.4297127D64EC6_8wekyb3d8bbwe\LocalCache\Local\runtime
    auto minecraftMSStorePath =
        FS::PathCombine(QFileInfo(partialPath).absolutePath(), "Local", "Packages", "Microsoft.4297127D64EC6_8wekyb3d8bbwe");
    minecraftMSStorePath = FS::PathCombine(minecraftMSStorePath, "LocalCache", "Local", "runtime");
    processpaths << minecraftMSStorePath;
```

this isn't exactly the perfect solution since these packages can change in the future, but as long as we keep it up to date this should be fine